.Lib9c.Tests/Action/ValidatorDelegation/DelegateValidatorTest.cs

```diff
@@ -259,7 +259,12 @@ public ValidatorInfo()
         public ValidatorInfo(Random random)
         {
             Balance = GetRandomFAV(DelegationCurrency, random);
-            Cash = GetRandomCash(random, Balance, 99);
+            do
+            {
+                Cash = GetRandomCash(random, Balance);
+            }
+            while (Cash.RawValue == Balance.RawValue);
+
```

Comment:
I couldn't find the intention of this change. Could you explain?

Reply:
When the cash is equle to the balance, exception is thrown

> I couldn't find the intention of this change. Could you explain?

Fixed because an exception was being thrown when Cash and Balance were equal. This was not clear, so I'll fix it again.